dtox.sh

```diff
@@ -7,40 +7,59 @@ ROOT="$(git rev-parse --show-toplevel)"
 BASE_INPUT=(
   "${ROOT}/docker/base/Dockerfile"
 )
-base_hash=$(cat ${BASE_INPUT[@]} | git hash-object -t blob --stdin)
+base_hash=$(cat "${BASE_INPUT[@]}" | git hash-object -t blob --stdin)
```

Comment:
If you haven't seen this script before, the hashing / labeling / short-circuiting are all to make `./dtox.sh` startup latency not too bad - at least on Linux:
```
$ hyperfine 'tox --version' './dtox.sh --version'
Benchmark 1: tox --version
  Time (mean ± σ):     105.1 ms ±   2.4 ms    [User: 94.7 ms, System: 10.2 ms]
  Range (min … max):   101.2 ms … 110.2 ms    28 runs

Benchmark 2: ./dtox.sh --version
  Time (mean ± σ):      1.223 s ±  0.041 s    [User: 0.025 s, System: 0.089 s]
  Range (min … max):    1.160 s …  1.284 s    10 runs

Summary
  'tox --version' ran
   11.64 ± 0.47 times faster than './dtox.sh --version'
```